README.md

```diff
@@ -0,0 +1,136 @@
+HighLine
+========
+
+by James Edward Gray II
```

Comment:
It was originally my solution to [a Ruby Quiz](http://rubyquiz.com/quiz29.html).  Greg added the menu system later, yes.

While I did a lot of work early on, HighLine 2 will be @abinoam's HighLine (and for the better)!  Your name should be right with mine at least, I would say.

I don't know exactly what we should make it look like yet.  Let's think on it and decide before we ship 2.0.